apps/frontend/src/utilities/tenable_util.ts

```diff
@@ -64,28 +65,70 @@ export class TenableUtil {
             resolve(response.request.finished);
           })
           .catch((error) => {
-            try {
-              if (error.code == 'ENOTFOUND') {
-                reject(
-                  `Host: ${this.hostConfig.host_url} not found, check the Host Name (URL) or the network`
-                );
-              } else if (error.response.data.error_code == 74) {
-                reject('Incorrect Access or Secret key');
-              } else {
-                reject(error.response.data.error_msg);
-              }
-            } catch (e) {
-              reject(
-                `Possible network connection blocked by CORS policy. Received error: ${error}`
-              );
-            }
+            reject(this.getRejectConnectionMessage(error));
           });
       } catch (e) {
         reject(`Unknown error: ${e}`);
       }
     });
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  getRejectConnectionMessage(error: any): string {
```

Comment:
We should have as robust error handling for Splunk as well.  Not a blocker for this PR.